README.md

```diff
@@ -16,6 +16,16 @@
 
 ## Quick Start
 
+Before running LEGO-WEBAPP, you have to compile LEGO-BRICKS
+
+```bash
+$ cd packages/lego-bricks
```

Comment:
You can build from the ropt as well, no? I believe you build both webapp and bricks then

Reply:
Ahh that's probably true, will check tomorrow